erdpy/wallet/pem.py

```diff
@@ -30,24 +31,55 @@ def parse(pem_file: Union[str, Path], index: int = 0) -> Tuple[bytes, bytes]:
     return seed, pubkey
 
 
-def parse_validator_pem(pem_file):
+def parse_validator_pem(pem_file, index: int = 0):
```

Comment:
add unit tests for the new methods that you added. take a look in folder `erdpy/tests`